src/components/app.tsx

```diff
@@ -176,20 +189,25 @@ export const App: React.FC<IAppProps<IModelInputState, IModelOutputState, IModel
           );
         }
 
-        setTrayObjects(newTrayObjects);
-        setShowTray(true);
+        setOutputStateAndSave({ trayObjects: newTrayObjects, showTray: true});
```

Comment:
If I remove this I still have the proper `fish` and `leafDecomposition` states when the sim completes.

Reply:
This appears to be true of all the original output state - `fish`, `algaeEaters`, `animalInstances`, `leafDecompositon`, and `leafEaters` - they are all reset to the initial values when we make this call.

Reply:
More specifically, they get set to the initial values defined in `initialOutputState`.

Reply:
Looks like I just forgot to merge in the original `outputState`. Fix coming shortly.

Reply:
Should be fixed now.